sda-commons-dependency-check/build.gradle

```diff
@@ -5,41 +5,50 @@ dependencies {
 
   // Add all new modules here to verify that there are no dependency clashes.
   api project(':sda-commons-client-jersey')
-  api project(':sda-commons-shared-wiremock-testing')
+  api project(':sda-commons-client-jersey-example')
```

Comment:
@JoergSiebahn : Do you know if there was a reason to not include the example modules?

Reply:
Because they are not provided for the consumers. But maybe they are a good indicator if dependency resolution will fail when the library is used in a service?